### PR TITLE
Feature search

### PR DESCRIPTION
**Pull request by sfrazier7**
Created the list and search functions

### CODE REVIEW

**Review by AnatalioNickles — CHANGES_REQUESTED** [review]
add a weird funny comment then lets merge! 1614

**Review by sushilshrestha1974 — COMMENTED** [review]
Reviewed

File: JavaLeague/src/Application.java [hunk 1]
```diff
@@ -1,26 +1,68 @@
...
+		//Loop to add contacts to created hashmap
+		for(int i = 0; i < contactList.size(); i++) {
+			hashContactList.put(i, contactList.get(i));
+		}
+		
+		Scanner key = new Scanner(System.in);
+		
+		//User Input
+		String query = key.nextLine();
+
```
Comment (AnatalioNickles) [inline JavaLeague/src/Application.java:32]: // this comment would go great here is you think it is appropriate

**Comment by BaileyPatt** [comment]
Looks Good

**Review by sushilshrestha1974 — COMMENTED** [review]
Reviewed

**Review by sushilshrestha1974 — COMMENTED** [review]
Reviewed

**Review by AnatalioNickles — APPROVED** [review]

**Pull request merged**